<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtLoginWidget</class>
 <widget class="QWidget" name="QtLoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QWidget" name="login_title" native="true">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#747678</string>
      </property>
      <property name="text">
       <string>创丰养殖</string>
      </property>
     </widget>
     <widget class="QPushButton" name="login_close_btn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>251</width>
        <height>361</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="login_logo">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>10</y>
          <width>128</width>
          <height>128</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="user_Edit">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>180</y>
          <width>171</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="password_Edit">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>240</y>
          <width>171</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>36</x>
          <y>210</y>
          <width>180</width>
          <height>1</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(240, 240, 240);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>36</x>
          <y>270</y>
          <width>180</width>
          <height>1</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(240, 240, 240);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="login_btn">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>290</y>
          <width>180</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLineEdit" name="port_Edit">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>120</y>
          <width>171</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="servce_ip">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>60</y>
          <width>171</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>36</x>
          <y>90</y>
          <width>180</width>
          <height>1</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(240, 240, 240);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>36</x>
          <y>150</y>
          <width>180</width>
          <height>1</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(240, 240, 240);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="config_ok_btn">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>190</y>
          <width>100</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="servce_set_btn">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
